flake.nix

```diff
@@ -0,0 +1,33 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+  outputs =
+    { nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (
+      system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+      in
+      {
+        devShells.default = pkgs.mkShell {
+          buildInputs = with pkgs; [
+            # generic
+            git
+            git-lfs
+            gnumake
+            gnutar
+
+            # frontend
+            nodejs_20
+
+            # backend
+            go_1_22
+            gofumpt
+            golangci-lint
```

Comment:
I think you should remove those last two, we build them from source in the Makefile. Can't have them unversioned here as they will eventually break if we lag behind in updates.